assets/tests/components/map.test.tsx

```diff
@@ -277,6 +278,33 @@ describe("<Map />", () => {
     await userEvent.click(mapRef.current!.getPane("mapPane")!)
 
     expect(openSpy).toHaveBeenCalled()
+    expect(window.FS!.event).toHaveBeenCalledWith(
+      "Dedicated street view toggled",
+      { streetViewEnabled: true }
+    )
+
+    expect(
+      screen.queryByRole("switch", { name: /Street View/, checked: false })
+    ).toBeInTheDocument()
+  })
+
+  test("turning off street view also fires a FullStory event", async () => {
+    mockFullStoryEvent()
+
+    const mapRef: MutableRefObject<LeafletMap | null> = { current: null }
+
+    render(
+      <Map vehicles={[]} allowStreetView={true} reactLeafletRef={mapRef} />
+    )
+
+    await userEvent.click(screen.getByRole("switch", { name: /Street View/ }))
+
+    await userEvent.click(screen.getByRole("switch", { name: /Street View/ }))
+
+    expect(window.FS!.event).toHaveBeenCalledWith(
+      "Dedicated street view toggled",
+      { streetViewEnabled: false }
```

Comment:
oh hey btw, the docs seem to say that they need a type suffix https://help.fullstory.com/hc/en-us/articles/360020623234

I've not seen the fullstory UI so I dunno what this does or looks like when you comply or not.
They also say that you can use camel case or underscores, and seem to suggest underscores on their [event api docs](https://help.fullstory.com/hc/en-us/articles/360020623274-FS-event-API-Sending-custom-event-data-into-FullStory#Examples), so we should probably standardize on camelCase or underscores.

Because the type suffix seems to need to start with an underscore, I think we should standardize on underscores.

Reply:
Ah, thanks for catching this! I'll update it to match their patterns.

Reply:
The docs at the first link you sent seem to have a mix of camelcase and snakecase, since camelcase is more in line with our variable naming on the rest of the Skate frontend I think I'll stick with that.

Reply:
Fixed in 0ddbf285!